app/javascript/groceries/components/DeletedItemToast.vue

```diff
@@ -0,0 +1,26 @@
+<template lang="pug">
+div
```

Comment:
(Note: as mentioned [here](https://github.com/davidrunger/david_runger/pull/4969/files#r1722724926), there must be only a single root component.)